<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-07T23:23:14 IST" name="Suite" finished-at="2023-10-07T23:24:57 IST" duration-ms="103287">
    <groups>
    </groups>
    <test started-at="2023-10-07T23:23:14 IST" name="Test" finished-at="2023-10-07T23:24:57 IST" duration-ms="103287">
      <class name="WalmartCaTest.OnboardTest">
        <test-method signature="VerifyStep1Link()[pri:1, instance:WalmartCaTest.OnboardTest@4dc27487]" started-at="2023-10-07T23:23:14 IST" name="VerifyStep1Link" finished-at="2023-10-07T23:24:18 IST" duration-ms="63445" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyStep1Link -->
        <test-method signature="BothValid()[pri:11, instance:WalmartCaTest.OnboardTest@4dc27487]" started-at="2023-10-07T23:24:18 IST" name="BothValid" finished-at="2023-10-07T23:24:26 IST" duration-ms="8621" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BothValid -->
        <test-method signature="VerifyNextButtonBytype()[pri:28, instance:WalmartCaTest.OnboardTest@4dc27487]" started-at="2023-10-07T23:24:26 IST" name="VerifyNextButtonBytype" finished-at="2023-10-07T23:24:54 IST" duration-ms="27518" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyNextButtonBytype -->
        <test-method signature="verifyDefaultProfile()[pri:41, instance:WalmartCaTest.OnboardTest@4dc27487]" started-at="2023-10-07T23:24:54 IST" name="verifyDefaultProfile" finished-at="2023-10-07T23:24:57 IST" duration-ms="3506" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test failded expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test failded expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at WalmartCaTest.OnboardTest.verifyDefaultProfile(OnboardTest.java:405)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDefaultProfile -->
      </class> <!-- WalmartCaTest.OnboardTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
